.gitignore

```diff
@@ -8,6 +8,7 @@
 *.user
 *.userosscache
 *.sln.docstates
+*.deb
```

Comment:
What is a .deb file?

Reply:
It's a Linux Debian file.  It's specific to my os and won't effect the actual project.  It popped into my branch, but I can remove it.

Reply:
I think its fine.